qtop.py

```diff
@@ -294,6 +297,15 @@ def create_job_counts(user_names, job_states, state_abbrevs):
         job_counts['cancelled_of_user'].setdefault(user_name, 0)
         job_counts['waiting_of_user'].setdefault(user_name, 0)
         job_counts['exiting_of_user'].setdefault(user_name, 0)
+        job_counts['restarting_of_user'].setdefault(user_name, 0)
+        job_counts['Eqw_of_user'].setdefault(user_name, 0)
+        job_counts['hold_of_user'].setdefault(user_name, 0)
+        job_counts['transferring_of_user'].setdefault(user_name, 0)
+        job_counts['threshold_reached'].setdefault(user_name, 0)
+        job_counts['job_ending_of_user'].setdefault(user_name, 0)
+        job_counts['suspended_of_user'].setdefault(user_name, 0)
+        job_counts['suspended_by_the_queue'].setdefault(user_name, 0)
```

Comment:
a little bit too many states and they are all hardwired here, which is not too great (as said, to be done later)